ShareX.UploadersLib/Forms/UploadersConfigForm.resx

```diff
@@ -13652,7 +13652,7 @@ Using an encrypted library disables sharing.</value>
     <value>13</value>
   </data>
   <data name="btnUpleaLogin.Text" xml:space="preserve">
-    <value>Login</value>
+    <value>Get API Key</value>
```

Comment:
@Jaex Figured this button label would be more representative of the action being performed rather than the previous label.